ci/scripts/connect-release-init.js

```diff
@@ -158,6 +163,16 @@ const initConnectRelease = () => {
             body: depsChecklist,
         });
     }
+
+    // Adding list of commits form the connect* packages to help creating and checking connect CHANGELOG.
+    const connectGitLog = getGitCommitByPackageName('connect*');
+
+    if (connectGitLog) {
+        comment({
+            prNumber,
+            body: connectGitLog,
```

Comment:
maybe something like (pseudo-code)
```
connectGitLog split by newlines into array
find index of record npm-release: @trezor/suite-connect 
keep only records between index 0 and found index.
```
this way we could see only those relevant changes? 

```

Reply:
That makes a lot of sense.
Added here https://github.com/trezor/trezor-suite/pull/8577/files#diff-5142b5e00105b89f50a738e522ed2024ed2f6c0f93ade8d94f052f22d7ddf822R174